contrib/apelscripts/condor_batch_blah.sh

```diff
@@ -52,6 +52,12 @@ do
         mv "$file" "$QUARANTINE_DIR"
         continue
     fi
+    # Check that APEL scaling information is available if requested
+    # Otherwise, quarantine the job information to allow manual fixing
+    if [[ -n $SCALING_ATTR && -z `condor_q -job $file -format "%f" "${SCALING_ATTR}"` ]]; then
```

Comment:
I could nitpick here about using `! -z $SCALING_ATTR` for consistency with previous checks, but this works just fine.